apps/gkyl_vlasov_priv.h

```diff
@@ -611,6 +669,19 @@ void vm_species_moment_calc(const struct vm_species_moment *sm,
 void vm_species_moment_release(const struct gkyl_vlasov_app *app,
   const struct vm_species_moment *sm);
 
+/** vm_species_emission API */
```

Comment:
Let's add the corresponding comments for the emission API here like the other private methods.